roles/barbican/tasks/main.yml

```diff
@@ -0,0 +1,97 @@
+---
+- name: Check if barbican user exists
+  action: shell getent passwd barbican
+  register: barbican_user
+  failed_when: False
+  changed_when: False
+
+- name: barbican user
+  user: name=barbican shell=/bin/false createhome=no
+  when: barbican_user|success
+
+- name: barbican user
+  user: name=barbican comment=barbican shell=/bin/false system=yes home=/nonexistent createhome=no
+  when: barbican_user|failed
+
+- name: /etc/barbican
+  file: dest=/etc/barbican state=directory owner=barbican group=barbican
+
+- name: barbican log dir
+  file: dest=/var/log/barbican state=directory mode=0755 owner=barbican group=adm
```

Comment:
Can we set the mode to 2750 so that newly created log files will be owned by the adm group?